doc/how_to/custom_components/reactive_html/reactive_html_layout.md

```diff
@@ -0,0 +1,272 @@
+)
+layout.servable()
+```
+
+You might notice that the values of `object1` and `object2` looks like they have been
+rendered as markdown! That is correct.
+
+Before inserting the value of a parameter in the
+`_template`, Panel transforms the value using `pn.panel`. And for a string value `pn.panel` returns
+a `Markdown` pane.
+
+Let's verify this.
+
+```{pyodide}
+print(type(layout.object1), type(layout.object2))
+```
+
+Lets for fun try another example
+
+```{pyodide}
+LayoutMultipleValues(
+    object1="Do you like **beat boxing**?",
+    object2="https://upload.wikimedia.org/wikipedia/commons/d/d3/Beatboxset1_pepouni.ogg",
+    styles={"border": "2px solid lightgray"},
+)
+```
+
+## Layout as literal `str` values
+
+If you want to show the *literal* `str` value of your parameter instead of the `pn.panel` return
+value you can configure that via the `_child_config` attribute.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutLiteralValues(pn.reactive.ReactiveHTML):
+    object1 = param.Parameter()
+    object2 = param.Parameter()
+
+    _ignored_refs = ("object1", "object2")
+    _child_config = {"object1": "literal", "object2": "literal"}
+
+    _template = """
+    <style>
+    .pn-container {height: 100%;width: 100%;}
+    </style>
+    <div class="pn-container">
+        <h1>Object 1</h1>
+        <div id="object1">${object1}</div>
+        <h1>Object 2</h1>
+        <div id="object2">${object2}</div>
+    </div>
+"""
+
+layout = LayoutLiteralValues(
+    object1="This is the **value** of `object1`", object2="This is the **value** of `object2`",
+    styles={"border": "2px solid lightgray"},
+)
+layout.servable()
+```
+
+Lets check the types
+
+```{pyodide}
+print(type(layout.object1), type(layout.object2))
+```
+
+## Layout a list of objects
+
+If you want to want to layout a list of objects you can use a *for loop*.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutOfList(pn.reactive.ReactiveHTML):
+    objects = param.List()
+
+    _template = """
+    <div id="container" class="pn-container">
+        {% for object in objects %}
+            <h1>Object {{ loop.index0 }}</h1>
+            <div id="object">${object}</div>
+            <hr/>
+        {% endfor %}
+    </div>
+"""
+
+LayoutOfList(objects=[
+            "I **love** beat boxing",
+            "https://upload.wikimedia.org/wikipedia/commons/d/d3/Beatboxset1_pepouni.ogg",
+            "Yes I do!"],
+        styles={"border": "2px solid lightgray"},
+).servable()
+```
+
+Please note that you must
+
+- wrap the `{% for object in objects %}` loop in an HTML element with an `id`. Here it is wrapped
+with `<div id="container">...</div>`.
+- close all HTML tags! `<hr>` is valid HTML, but not valid with `ReactiveHTML`. You must close it
+as `<hr/>`.
+
+Please note you can optionally
+
+- get the index of the `{% for object in objects %}` loop via `{{ loop.index0 }}`.
+
+## Create a list like layout
```

Comment:
I notice that you're using jinja template, but when I tested this and updated the objects parameter, this actually updated--which is awesome. I guess that's some magic from NamedListLike? Might be worth mentioning this is the case though

Reply:
I've tried to mention this.